nest-cli.json

```diff
@@ -0,0 +1,19 @@
+
+{
+  "collection": "@nestjs/schematics",
+  "monorepo": true,
+  "compilerOptions": {
+    "webpack": false
+  },
+  "projects": {
+    "ocean-api": {
+      "type": "application",
```

Comment:
hmm, I am wondering if we can forgo `nest-cli.json` and just use the barebone setup.